index.md

```diff
@@ -29,22 +18,29 @@ Along with bug fixes and general improvements, Anbox Cloud 1.13 comes with:
 * Direct rendering for Intel and AMD GPUs
 * OpenGL ES 3.2 support
 
-|  |  |
-|--|--|
-| [Release notes](https://discourse.ubuntu.com/t/release-notes/17842) | All new features, improvements and bug fixes |
-| [Release roadmap](https://discourse.ubuntu.com/t/release-roadmap/19359) | Planned updates and features for upcoming releases |
+For detailed information, see the [Release notes](https://discourse.ubuntu.com/t/release-notes/17842).
+
+## Project and community
+
+Anbox Cloud is a Canonical product.
```

Comment:
Should we call out here that Anbox Cloud is not open source and has no link to the open source Anbox?

Reply:
Yes, for sure!
I added `It is closed source and has no link to the Anbox open source project.`
A bit unsure if we should add a link to Anbox - we have it in the glossary, but I think it would be misleading here on the start page.